examples/app-basic/cypress/integration/01-app-config.spec.ts

```diff
@@ -0,0 +1,26 @@
+import { e2e } from '@grafana/e2e';
+import { testIds } from '../../src/components/testIds';
+
+const { appConfig } = e2e.getSelectors(testIds);
+
+describe('configurating app', () => {
+  beforeEach(() => {
+    cy.visit('http://localhost:3000/plugins/grafana-basic-app');    
```

Comment:
We should be able to improve this with a flow to e2e package or some such.

Reply:
I totally agree! Lets add it to the list to expose some kind of function that will fetch this based on the environment we are running the tests in.